### DISCUSSION TITLE
sort search results

### DISCUSSION THREAD

**Discussion by nagylzs** [doublecmd/doublecmd#1390] [Suggestions]
File search results (Meta+F7) are always shown in arbitrary order. There should be a way to specify the sort order for the results. At least these: file path, file name, creation / last modification time, size.

**Comment by JonnyTech** [comment]
Press the "Feed to listtbox" button then you order or manipulate the results as you please.

**Comment by nagylzs** [comment]
Partly works, but I actually need to sort them by file path. For example, I search for file name "001.sql" and find it in several directories. I feed into listbox, and then I can sort it by file name, but not file path. The "file path" column is not available, only the name. But all file names are the same.

**Comment by nagylzs** [comment]
Well, I can see that I can add DC().GETFILEPATH() to a column config. But for some reason, it displays the directory of the file, and not the full file path. Is there a way to display (and sort) full file path? Including both the directory AND the file name?

To me it seems strange that "GETFILEPATH" returns the directory path, and not the full file path.
